modell/src/main/kotlin/no/nav/tilbakekreving/entities/BrukerEntity.kt

```diff
@@ -0,0 +1,28 @@
+package no.nav.tilbakekreving.entities
+
+import kotlinx.serialization.Serializable
+import no.nav.tilbakekreving.kontrakter.bruker.Kjønn
+import no.nav.tilbakekreving.kontrakter.bruker.Språkkode
+import no.nav.tilbakekreving.person.Bruker
+import java.time.LocalDate
+
+@Serializable
+data class BrukerEntity(
+    val ident: String,
+    var språkkode: String? = null,
+    var navn: String? = null,
+    var fødselsdato: String? = null,
+    var kjønn: String? = null,
+    var dødsdato: String? = null,
+) {
+    fun fraEntity(): Bruker {
+        return Bruker(
+            ident = ident,
+            språkkode = språkkode?.let { Språkkode.valueOf(it) },
+            navn = navn,
+            fødselsdato = fødselsdato?.let { LocalDate.parse(fødselsdato!!) },
```

Comment:
`?.let` sjekker vel at fødselsdato eksiterer og trenger vel ikke double bang `!!`
```suggestion
            fødselsdato = fødselsdato?.let { LocalDate.parse(fødselsdato) },
```

Reply:
Man kan bruke `it` i stedet, men optimalt sett vil det være en LocalDate i DTOen